test/suite/wttest.py

```diff
@@ -830,6 +830,21 @@ def recno(self, i):
         """
         return i
 
+@contextmanager
+def open_cursor(session, uri: str):
+    """
+    Open a cursor instance that supports 'with' statements.
+    """
+    assert session is not None
```

Comment:
I think we could simply do:
```suggestion
    assert session, 'Session has to be valid'
    assert uri, 'The URI has to be valid and non-empty'
```

Reply:
These are better assertions -- thanks. In considering the change I wonder if they should be removed altogether as the arguments are simply passed to `session.open_cursor`. What do you think?

Reply:
Yes, happy to remove them. If the arguments are wrong, we wont go much further anyway!